src/Sylius/Behat/Context/Setup/ProductReviewContext.php

```diff
@@ -58,13 +59,52 @@ public function __construct(
      */
     public function productHasAReview(ProductInterface $product)
     {
+        $review = $this->createProductReview($product, 'Title', 5);
+
+        $this->productReviewRepository->add($review);
+    }
+
+    /**
+     * @Given /^(this product) has(?:| also) a review titled "([^"]+)" and rated (\d+) with a comment "([^"]+)" added by (customer "[^"]+")$/
+     */
+    public function thisProductHasAReviewTitledAndRatedWithACommentAddedByCustomer(
+        ProductInterface $product,
+        $title,
+        $rating,
+        $comment,
+        CustomerInterface $customer
+    ) {
+        $review = $this->createProductReview($product, $title, $rating, $comment, $customer);
+
+        $this->productReviewRepository->add($review);
+    }
+
+    /**
+     * @param ProductInterface $product
+     * @param string $title
+     * @param int $rating
+     * @param string|null $comment
+     * @param CustomerInterface|null $customer
+     *
+     * @return ReviewInterface
+     */
+    private function createProductReview(
+        ProductInterface $product,
+        $title,
+        $rating,
+        $comment = null,
+        CustomerInterface $customer = null
```

Comment:
We should not be able to add a review without customer, this should be prevented on a database level also.